src/openrct2/CmdlineSprite.cpp

```diff
@@ -220,8 +220,8 @@ static GamePalette GetPaletteFromString(std::string_view paletteOption, std::str
     }
     else if (paletteOption != "openrct2")
     {
-        std::string finalPath{ palettePath };
-        finalPath.append(paletteOption);
+        std::string finalPath;
+        finalPath.append(palettePath).append("/").append(paletteOption);
```

Comment:
This doesn't seem safe cross-platform, aren't there some path related helper functions?

Reply:
C++ has [`std::filesystem::path`](https://en.cppreference.com/w/cpp/filesystem/path). Include `core/FileSystem.hpp` for this.

Reply:
None of the cmdline code is safe cross platform.